dali/test/python/test_operator_one_hot.py

```diff
@@ -121,27 +126,28 @@ def check_one_hot_operator(source, axis=-1, expected_output_dim=None, axis_name=
 
 def test_one_hot_scalar():
     np.random.seed(42)
-    for i in range(10):
-        yield partial(check_one_hot_operator, random_scalars_batch, axis_name='O')
+    for device in ['cpu', 'gpu']:
+        for i in range(10):
```

Comment:
I know it was not added during this PR, by why do we have some for loops creating the same test 10 times? Maybe I am missing something?

Reply:
I guess the point is to test on a few "random" examples not just a single one, random_scalars_batch returns different examples each time.